src/data-visualisation/power-bi/learning/certifications/AI-exams/ai-102.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: page
+title: "AI-102"
+description: "Overview of the AI-102 exam"
+status: DRAFT
+tags: ai-certs
+order: 10
+---
+# Designing and Implementing a Microsoft Azure AI Solution Exam AI-102
```

Comment:
Heading level looks wrong here should be `##`?

Reply:
I set these all to # so they are larger, but can change to ## if necessary?